http/README.md

```diff
@@ -41,29 +41,32 @@ Nodes can also link to a specific resource directly, similar to how a URL includ
 
 ## Namespace
 
-libp2p does not squat the global namespace. By convention, all libp2p services are located at a well-known URL: `https://example.com/.well-known/libp2p/<service name>/<path (optional)>`.
+libp2p does not squat the global namespace. By convention libp2p services can be discovered by accessing the configuration file at a well-known URL (RFC 8615): `.well-known/libp2p.json` (e.g. https://example.com/.well-known/libp2p.json). This allows server operators to dynamically change the URLs of the services offered, and to not hard-code any assumptions how a certain resource is meant to be interpreted.
```

Comment:
That makes sense. What would happen though if the client doesn’t set this header? Would we just return a 4xx error?

Reply:
We can treat a missing `Accept` header as "I don't care what you serve me". A well-behaved client should set them but in principle, the client can't force the server to send a certain representation. A well-behaved server might respond with 415 if it doesn't have the asked representation but I've found that especially via APIs, servers only support one content type anyway and they mostly ignore `Accept`.

My point is mostly that we should not be using "file extensions" in URLs because resources are more abstraction than file systems. HTTP's way of describing data types is through the `Content-Type` header.